src/main/groovy/edu/oregonstate/mist/locations/frontend/db/LocationDAO.groovy

```diff
@@ -254,6 +256,18 @@ class LocationDAO {
             query.must(parkingZoneGroupQuery)
         }
 
+        if (ada) {
+            query.must(QueryBuilders.rangeQuery("attributes.adaParkingSpaceCount").gt(ada))
```

Comment:
What do you think making this a greater than or equal to instead? If I want parking lots with 1 or more ada parking spaces (or moto or ev, etc.) then I would need to put in an `ada` parameter of 0, but that returns every location (ones with `null` for the ada space count too).